decompiler/test/ff7_field_decomp_md1_1_test.cpp

```diff
@@ -0,0 +1,38 @@
+#include <gmock/gmock.h>
+
+#include "decompiler/ff7_field/ff7_field_disassembler.h"
+#include "decompiler/ff7_field/ff7_field_engine.h"
+#include "decompiler/ff7_field/ff7_field_codegen.h"
+
+#include "control_flow.h"
+#include "util.h"
+#include "graph.h"
+#include "make_unique.h"
+#include "sudm.h"
+#include "lzs.h"
+#include "ff7_field_dummy_formatter.h"
+
+TEST(FF7Field, DISABLED_Decomp_MD1_1)
+//TEST(FF7Field, Decomp_MD1_1)
+{
+    std::cout << "ready" << std::endl;
```

Comment:
Each of these are the same apart from the file name, could push them into a helper func